src/components/app/list/types.tsx

```diff
@@ -150,3 +150,10 @@ export interface ManifestUrlList {
     pointsTo: string
     urls: string[]
 }
+
+export interface CopyToClipboardTextProps {
+    text: string
+    rootClassName?: string
+    iconClass?: string
+    placement?: any
```

Comment:
replace 'any' by the types which takes four placement i.e bottom , top etc , get refrence from the component inside

Reply:
Change was made by Sohel.

Reply:
This should have been replaced with the `Placement` type but initially was kept to `any`. We will update this.